pact/yaml/testnet/swap-callable.yaml

```diff
@@ -0,0 +1,13 @@
+codeFile: ../swap-callable.pact
```

Comment:
This is fine, but I think I'd like to look at a pact upgrade that can concat files, so I might get that together today ...